pkg/certificate/fake_manager.go

```diff
@@ -18,7 +19,9 @@ var (
 	validity = time.Hour
 )
 
-type fakeMRCClient struct{}
+type fakeMRCClient struct {
```

Comment:
Can we get rid of this whole thing? There's one place where it's used, which can be easily switched to the `tresorFake.NewFake`

Reply:
I believe using `tresorFake` would result in a circular dependency.

Reply:
but the usages (there's now 2) of this is outside of the package. Can we get rid of this, and migrate the usage of FakeCertManager to tresor.NewFake?

Reply:
It is used in `manager_test.go` which is in this package.